wgpu/tests/clear_texture.rs

```diff
@@ -239,20 +239,28 @@ fn single_texture_clear_test(
     // TODO: Read back and check zeroness?
 }
 
+fn round_up(value: u32, alignment: u32) -> u32 {
+    ((value + alignment - 1) / alignment) * alignment
+}
+
 fn clear_texture_tests(
     ctx: &TestingContext,
     formats: &[wgpu::TextureFormat],
     supports_1d: bool,
     supports_3d: bool,
 ) {
     for &format in formats {
+        let desc = format.describe();
+        let rounded_width = round_up(64, desc.block_dimensions.0 as u32);
+        let rounded_height = round_up(64, desc.block_dimensions.1 as u32);
```

Comment:
why are we rounding?
WebGPU allows creating any texture sizes regardless of the block size

Reply:
What how does that work with vk requires textures a multiple if the block size?

Reply:
Per https://gpuweb.github.io/gpuweb/#dom-gpudevice-createtexture at least currently, images need to be the block size.

> descriptor.[size](https://gpuweb.github.io/gpuweb/#dom-gputexturedescriptor-size).[width](https://gpuweb.github.io/gpuweb/#extent3d-width) must be multiple of [texel block width](https://gpuweb.github.io/gpuweb/#texel-block-width).

> descriptor.[size](https://gpuweb.github.io/gpuweb/#dom-gputexturedescriptor-size).[height](https://gpuweb.github.io/gpuweb/#extent3d-height) must be multiple of [texel block height](https://gpuweb.github.io/gpuweb/#texel-block-height).

Reply:
oh right, hmm